TunnelVisionLabs.ReferenceAssemblyAnnotator/TunnelVisionLabs.ReferenceAssemblyAnnotator.props

```diff
@@ -14,4 +14,10 @@
     </AnnotateReferenceAssembliesDependsOn>
   </PropertyGroup>
 
+  <PropertyGroup Condition="
+                 ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' AND '$(TargetFrameworkVersion.Substring(1))' &gt;= '3.0')
+                 OR ('$(TargetFrameworkIdentifier)' == '.NETStandard' AND '$(TargetFrameworkVersion.Substring(1))' &gt;= '2.1')">
```

Comment:
📝 These will be problematic in the **.props** file. The group should be moved to the **.targets** file where the default value for the property is initialized.

Reply:
Updated.

Reply:
Tests show that this isn't final, working on it.

Reply:
Oops, lost the `!`.

Reply:
Updated and verified working across all listed target frameworks.